apollo-router/src/state_machine.rs

```diff
@@ -226,8 +226,19 @@ impl<FA: RouterSuperServiceFactory> State<FA> {
                         &mut guard,
                         signals,
                     )
-                    .await
-                    {
+                    .await;
+
+                    u64_counter!(
+                        "apollo_router_reload",
```

Comment:
```suggestion
                        "apollo.router.reload",
```
Let's use `.` rather than underscore as this is in line with otel standards.